modules/oxide-components/src/main/ts/bespoke/tinyai/bubbles/UserPromptBubble.tsx

```diff
@@ -0,0 +1,18 @@
+import type { FunctionComponent } from 'react';
+
+import { classes } from '../../../utils/Styles';
+
+interface UserPromptProps {
+  readonly prompt: string;
+}
+
+export const UserPromptBubble: FunctionComponent<UserPromptProps> = ({ prompt }) => {
+  return (
+    <div
+      tabIndex={-1}
+      className={classes([ 'tox-ai__user-prompt__text' ])}
```

Comment:
I'd rather we use the Bem helper function here instead of the `classes` helper function